Update settings.md (#5351)
segmentio/segment-docs: src/engage/settings.md

To add a Destination, select the **+ Add destination** button, or navigate to **Connections > Destinations** within your Segment workspace. To learn more about sending Engage information to Segment Destinations, view the [Using Engage Data](/docs/engage/using-engage-data/) documentation.

> You can delete a Destination from the Destinations tab in Engage settings (**Engage > Engage settings > Destinations**). 

Reviewer: ```suggestion
You can delete a Destination from the Destinations tab in the Engage settings (**Engage > Engage settings > Destinations**). 
```

> To delete a Destination, navigate to **Engage > Engage Settings > Destinations**. You cannot do this via Connections > Destinations.

Reviewer: ```suggestion
You can delete a Destination from the Destinations tab in Engage settings (**Engage > Engage settings > Destinations**). 
```
